server/src/main.py

```diff
@@ -1,13 +1,9 @@
-from fastapi import FastAPI
-from starlette.middleware.cors import CORSMiddleware
+import os
 
-application = FastAPI()
+import uvicorn
 
-
-application.add_middleware(CORSMiddleware, allow_origins=["*"], allow_credentials=True, allow_methods=["*"],
-                           allow_headers=["*"])
-
-
-@application.get("/")
-async def root():
-    return {"message": "Hello World"}
+if __name__ == "__main__":
```

Comment:
Running the app like this instead of using uvicorn from the command enables more control over config values passed into uvicorn in the future. This helps with possible logging and CI/CD -implementations.